nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -562,6 +571,20 @@ in {
           The list of identity servers trusted to verify third party identifiers by this server.
         '';
       };
+      account_threepid_delegates_email = mkOption {
```

Comment:
Is that an elegant way to this ? @pacien or @Ma27

Reply:
No, those should be nested using a submodule type: `account_threepid_delegates.email` and `account_threepid_delegates.msisdn`. Take a look at how [`listeners.*` are defined](https://github.com/NixOS/nixpkgs/blob/e2664d0e942236f3244c1e8cefe8c38b8bcb6e29/nixos/modules/services/misc/matrix-synapse.nix#L221).

Reply:
(currently working on it btw).

Reply:
It should still be `account_threepid_delegates.email` instead of `account_threepid_delegates_email`. Same for `msisdn`.

Reply:
You're right. Will fix this later today :)

Reply:
Oof, forgot about this, will take care of this tonight :)

Reply:
Addressed in the latest commit :)